include/cadmium/logger/dynamic_common_loggers.hpp

```diff
@@ -27,17 +27,37 @@
 #ifndef CADMIUM_DYNAMIC_COMMON_LOGGERS_HPP
 #define CADMIUM_DYNAMIC_COMMON_LOGGERS_HPP
 
-#include <sstream>
-#include <cadmium/modeling/dynamic_coupled.hpp>
+#include <cadmium/engine/common_helpers.hpp>
 
 namespace cadmium {
     namespace dynamic {
         namespace logger {
 
+            struct routed_messages {
+                std::string from_port;
+                std::string to_port;
+                std::vector<std::string> from_messages;
+                std::vector<std::string> to_messages;
+
+                routed_messages() = default;
```

Comment:
Is =default the expected behavior? or should it be =deleted?

Reply:
Done, not sure if it is necessary to restrict this struct to be fully constructed, but since I don't use the default constructor I've deleted it as you suggested.

Reply:
I realized that I need the default constructor, so I rollback your suggestion to let original = default() version

Reply:
I finished by implementing a constructor for the empty case that defines the ports and deleted the default constructor.